clients/drcachesim/common/memtrace_stream.h

```diff
@@ -240,6 +260,17 @@ class memtrace_stream_t {
     {
         return false;
     }
+
+    /**
+     * Returns the value of the specified statistic for this output stream.
+     * The values for all output stream must be summed to obtain global counts.
+     * Returns -1 if statistics are not supported for this stream.
+     */
+    virtual int64_t
```

Comment:
OK, but now all my sanity checks and unit tests have to use almost-equals-within-epsilon instead of `==`?  Should they assume `==` will work for integers that fit in the 54-bit mantissa?

Reply:
For stats that are integral counts, using `==` and expecting exact match seems entirely reasonable to me.  If we have integral stats where we need >54 bits yet don't guard against int64 overflow in the scheduler, I feel like we're already in the danger zone.  At least with double any overflow won't wrap.

Reply:
Done.